src/objects.js

```diff
@@ -1,43 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
-};
+  return {
+    'name': name,
+    'age': age
+  }
+}
 
 const getName = object => {
-  // your code here
+  return object.name; 
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+ return person.age > 65 ? true : false; 
 };
 
 const getAges = people => {
-  // your code here
+ return people.map(person => person.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  const agesArray = people.map(person => person.age);
+  const agesTotal = agesArray.reduce((accumulator, age) => accumulator + age);
+  return agesTotal / people.length;
+
+  
 };
 
-const createTalkingPerson = (name, age) => {
-  // your code here
+const createTalkingPerson = (name, age) => { 
+  return {
+    'name' : name,
+    'age' : age,
+    'introduce' : otherName => {
+      return `Hi ${otherName}, my name is ${name} and I am ${age}!`;
+    }
+    }
+ 
```

Comment:
Just indentation and the same issue with the keys for objects as mentioned above here
```
  return {
    name,
    age,
    'introduce' : otherName => {
      return `Hi ${otherName}, my name is ${name} and I am ${age}!`;
    }
  }
```